mep/common/admin.py

```diff
@@ -50,8 +52,11 @@ def group_names(self, obj):
 
 
 class ImportExportModelResource(ModelResource):
-    def __init__(self, *x, **y):
-        super().__init__(*x, **y)
+    max_objects_to_index = 100
```

Comment:
I'm curious how you arrived at this number, wondering if we could go a bit higher

Reply:
indexing 6000 records took 3 minutes I believe, so 1000 records would take 30sec, which is maybe our request limit?, so I just went an order of magnitude below that for now. We can probably increase it